cliquet_docs/contributing.rst

```diff
@@ -143,7 +143,7 @@ Step 1
      $ git checkout -b prepare-X.Y.Z
      $ prerelease
      $ vim docs/conf.py
-     $ rm -rf .venv
+     $ make maintainer-clean
```

Comment:
I am not a big fan of this to update the requirements.

Since this removes the virtualenv, we have to re-install `pip install "zest.releaser[recommended]"` otherwise the wheel won't be published on pypi.